deployment/docker/Dockerfile

```diff
@@ -12,17 +12,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-ARG OS=linux
 ARG ARCH=amd64
```

Comment:
Do we still need these 2 args?

Reply:
`ARG OS=linux` is not needed, so deleted. `ARG ARCH=amd64` is required because it's used in `FROM k8s.gcr.io/debian-base-${ARCH}:0.4.1` instruction.